src/trading/postLimitOrder.ts

```diff
@@ -9,29 +9,36 @@ export async function postLimitOrder(
   params: LimitOrderParameters,
   advancedSettings?: LimitOrderAdvancedSettings,
   _orderBookApi?: OrderBookApi
-): Promise<string> {
-  const {
-    appCode,
-    chainId,
-    sellToken,
-    buyToken,
-    sellAmount,
-    buyAmount,
-    partnerFee,
-    slippageBps = 0,
```

Comment:
We set a default value here, but `params` goes further in other functions.
Because of that I mutate `params` in order to set default values